content/_index.md

```diff
@@ -1,10 +1,32 @@
 ---
 title: "OpenMDM - Measured Data Management"
 seo_title: "OpenMDM - Measured Data Management"
-headline: "OpenMDM - Measured Data Management"
-tagline: "OpenMDM - Measured Data Management"
+headline: "Plan. Measure. Share."
+tagline: "And simplify your measured data management."
+jumbotron_tagline_class: "col-sm-18"
 description: "openMDM® 5 is a kit of components and concepts, which can be used to compose applications for measured data management systems. The development of components and concepts is driven by the openMDM® Eclipse Working Group. All components and concepts are published under the Eclipse Public License."
 date: 2020-09-08T11:00:00-04:00
 layout: "single"
 hide_breadcrumb: true
----
\ No newline at end of file
+hide_sidebar: true
+hide_page_title: true
+container: "container-fluid"
+links: [[href: "https://accounts.eclipse.org/mailing-list/open-measured-data-wg", text: "Join Our Mailing List"], [href: "https://wiki.eclipse.org/Open-Measured-Data-Management-WG#Steering_Committee", text: "Learn More"]]
+---
+
+{{< grid/section-container >}}
+  {{< grid/div class="col-sm-15 margin-top-30" isMarkdown="false" >}}
+    {{< newsroom/news
+          title="News"
+          titleClass="heading-line text-center"
+          id="news-list-container"
+          publishTarget="openmdm"
+          count="5"
+          includeList="true" >}}
+  {{</ grid/div >}}
+  {{< grid/div class="col-sm-8 col-sm-offset-1 margin-top-30" isMarkdown="false" >}}
+    {{< events >}}
```

Comment:
Why make/use a custom shortcode when one exists in the base theme?

Reply:
I was gonna to post a question about this.

The default {{< newsroom/events >}} shortcode will generate the events horizontally aligned, I wonder if there is a way I can pass params or set configs to make this shortcode to align events vertically?

Reply:
See above answer, you would change the template